app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,271 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
+
+p,
+h1,
+h2,
+h3,
+h4 {
+  font-family: "Lobster", cursive;
+}
+
+a { text-decoration: none; }
+
+:root {
+  --primary-color: #3778c2;
+  --secondary-color: #5fb523;
+  --text-color: #434b54;
+}
```

Comment:
Good job working on the project requirements. But you used the wrong font for this project, it says in the project requirements that:
Design


The student should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:


Colors
Typography: font face, size and weight.
Layout: composition and space between elements.

So in the case the correct font you are supposed to use is :

For the Heading the correct font to use is Proxima Nova Bold

Then for the general text the correct font to use is Proxima Nova Light

Kindly look at the screen shot below it shows the design rules for the font just incase you feel Im being unfair:

![Screenshot 2023-07-13 at 4 59 17 PM](https://github.com/Hisoka37/Budget_App/assets/79506105/079404e4-3e30-4498-af86-4715341c0f28)

It is important to pay attention to detail.

Reply:
@KayLemba , I couldn't find that font, I will try to use one close to it.